backend/python/app/services/implementations/story_service.py

```diff
@@ -87,6 +92,27 @@ def import_story(self, details, file):
             self.logger.error(str(error))
             raise error
 
+    def export_story_translation(self, id):
+        story_details = self.get_story_translation(id=id)
+        doc = docx.Document()
+        doc.add_heading(story_details["title"])
+
+        for line in story_details["translation_contents"]:
+            doc.add_paragraph(line["translation_content"])
+
+        upload_folder_path = os.getenv("UPLOAD_PATH")
+
+        upload_path = os.path.join(
+            upload_folder_path, secure_filename("downloadable-story-translation.docx")
+        )
+        doc.save(upload_path)
+        file_dto = FileDTO(path=upload_path)
+        self.logger.info(file_dto)
+        new_file = File(**file_dto.__dict__)
```

Comment:
hmmmm is it necessary to make a file table entry if it's going to be deleted? is this here for now for ease of testing?

Reply:
added it so i could use the get_file function since it uses db query to get the file

Reply:
hmmm i see, mind extracting out the relevant parts of `get_file` to a different function so you can call that directly? That way we can eliminate unnecessary DB commits